tests/test_drydep.f90

```diff
@@ -0,0 +1,185 @@
+      MetState%AIRDEN(i) = 1.2   ! kg/m3
+      MetState%ZMID(i) = (MetState%NLEVS*100 - I*100)   ! m
+   end do
+
+   DryDepState%SchemeOpt = 1
+   ! Turn off resuspension
+   DryDepState%Resuspension = .FALSE.
+
+   ! Allocate DiagState
+   call cc_allocate_diagstate(Config, DiagState, ChemState, RC)
+   if (rc /= CC_SUCCESS) then
+      errMsg = 'Error in cc_allocate_diagstate'
+      stop 1
+   endif
+
+   title = "DryDep Test 2 | Test GOCART DryDep defaults"
+
+   call cc_drydep_init(Config, DryDepState, ChemState, rc)
+   if (rc /= CC_SUCCESS) then
+      errMsg = 'Error in cc_drydep_init'
+      call cc_emit_error(errMsg, rc, thisLoc)
+      stop 1
+   end if
+
+   ! commenting out for now
+   call cc_drydep_run(MetState, DiagState, DryDepState, ChemState, rc)
+   if (rc /= CC_SUCCESS) then
+      errMsg = 'Error in cc_drydep_run'
+      call cc_emit_error(errMsg, rc, thisLoc)
+      stop 1
+   end if
+
+   !write (*,*) ' - call print_info for ', title, ' - '
+   call print_info(Config, DryDepState, MetState, title)
+   !write (*,*) ' - finished print_info for ', title, ' - '
+   call assert(DiagState%drydep_frequency(1) > 0.0_fp, "Test GOCART DryDep Scheme (no resuspension)")
+   !write (*,*), '-- '
+   !write (*,*), 'Completed ', title
+   !write (*,*), '--'
+
+
+   !----------------------------
+   ! Test 3
+   !----------------------------
+   title = "drydep Test 3 | resuspension is .TRUE. "
+   ChemState%nSpeciesAerodrydep = 1
+   ! Turn on resuspension
+   DryDepState%Resuspension = .TRUE.
+   DryDepState%particleradius = 0.000001   ! [m]
+   DryDepState%particledensity = 2500.   !  [kg/m3]
+
+   call cc_drydep_run(MetState, DiagState, DryDepState, ChemState, rc)
+   if (rc /= CC_SUCCESS) then
+      errMsg = 'Error in cc_drydep_run'
+      call cc_emit_error(errMsg, rc, thisLoc)
+      stop 1
+   end if
+
+   call print_info(Config, DryDepState, MetState, title)
+   call assert(DiagState%drydep_frequency(1) > 0.0_fp, "Test 2 GOCART drydep Scheme (resuspension activated)")
+
+
+
+contains
+
+   subroutine print_info(Config_, DryDepState_, MetState_, title_)
+
+      type(ConfigType), intent(in) :: Config_
+      type(MetStateType), intent(in) :: MetState_
+      type(DryDepStateType), intent(in) :: DryDepState_
+      character(len=*), intent(in) :: title_
+
+      write(*,*) '======================================='
+      write(*,*) title_
+      write(*,*) '======================================='
+      write(*,*) '*************'
+      write(*,*) 'Configuration '
+      write(*,*) '*************'
+      write(*,*) 'Config%drydep_activate = ', Config_%drydep_activate
+      write(*,*) 'Config%drydep_scheme = ', Config_%drydep_scheme
+      write(*,*) 'Config%drydep_resuspension = ', Config_%drydep_resuspension
+
+      if (DryDepState%Activate) then
+
+         write(*,*) 'DryDepState%Activate = ', DryDepState_%Activate
+         write(*,*) 'DryDepState%SchemeOpt = ', DryDepState_%SchemeOpt
```

Comment:
Suggest adding `DryDepState_%Resuspension` as well

Reply:
I'm sorry - I think I resolved some of these too soon and without address the change!  I clicked too fast!!

Reply:
Feel free to unresolve if you find one

Reply:
Added.